src/jdk.internal.vm.ci/share/classes/jdk/vm/ci/hotspot/HotSpotConstantPool.java

```diff
@@ -581,8 +584,8 @@ private static String argumentAsString(JavaConstant arg) {
     }
 
     @Override
-    public BootstrapMethodInvocation lookupBootstrapMethodInvocation(int rawCpi, int opcode) {
-        int cpi = opcode == -1 ? rawCpi : rawIndexToConstantPoolIndex(rawCpi, opcode);
+    public BootstrapMethodInvocation lookupBootstrapMethodInvocation(int index, int opcode) {
+        int cpi = opcode == -1 ? index : indyIndexConstantPoolIndex(index, opcode);
```

Comment:
Why would opcode be -1 here?

Reply:
So that tests such as TestDynamicConstant.java can [iterate through the constant pool, looking for invokedynamic related entries](https://github.com/openjdk/jdk/blob/f143380d013b8c0e5ab7ca0026c34e27e7946f69/test/hotspot/jtreg/compiler/jvmci/jdk.vm.ci.hotspot.test/src/jdk/vm/ci/hotspot/test/TestDynamicConstant.java#L356-L359).